plugins/tech-radar/src/components/RadarComponent.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState, FC } from 'react';
+import { Progress, useApi, errorApiRef, ErrorApi } from '@backstage/core';
+import Radar from '../components/Radar';
+import { TechRadarComponentProps, TechRadarLoaderResponse } from '../api';
+import getSampleData from '../sampleData';
+
+const useTechRadarLoader = (props: TechRadarComponentProps) => {
```

Comment:
This entire function can be replaced with a https://github.com/streamich/react-use/blob/master/docs/useAsync.md

Reply:
Oh, of course there is! Definitely need to read the docs for `react-use` more regularly :P